_episodes/03-packaging-installing.md: moving Pip section.

vinisalazar opened the pull request:
Related to #77.

Summary of changes:
 - Moves the Pip section further up in the episode.

Edit: I also cherry-picked 041a952 into this branch to prevent conflicts.

Best,
Vini

vinisalazar committed:
_episodes/03-packaging-installing.md: capitalizes occurrences of Python as a noun.

vinisalazar committed:
_episodes/03-packaging-installing.md: moving Pip section. Related to #77.

brownsarahm commented on _episodes/03-packaging-installing.md:
```diff
@@ -52,7 +52,7 @@ Keep motivation as easing in  and gradually scaling up to stay inclusive -->
 ## Pip
 
-Pip is the most common package manager for python. Pip allows you to easily install python packages locally from your computer or from an online repository like the [Python Package Index (PyPI)](https://pypi.org/). Once a package is installed with pip, you can `import` that package and use it in your own code.
+Pip is the most common package manager for Python. Pip allows you to easily install Python packages locally from your computer or from an online repository like the [Python Package Index (PyPI)](https://pypi.org/). Once a package is installed with pip, you can `import` that package and use it in your own code.
```
is this in two places right now?

vinisalazar replied:
My bad, that was a bit of an imperfect merge. I pushed ffb1ff7 that fixes it.

vinisalazar committed:
_episodes/03-packaging-installing.md: Delete repeated Pip section.
  - Code review for #79.

brownsarahm commented on _episodes/03-packaging-installing.md:
```diff
@@ -370,24 +288,108 @@ from a local directory.
 
+## Pip
+
+Pip is the most common package manager for python. Pip allows you to easily install python packages locally from your computer or from an online repository like the [Python Package Index (PyPI)](https://pypi.org/). Once a package is installed with pip, you can `import` that package and use it in your own code.
```
pypi is mentioned above when introducing setup.py

I think that's why i initially had this section where it is currently

vinisalazar replied:
Thanks for noticing this.

I pushed a5be1e3 that edits the phrasing to properly introduce Pip.

vinisalazar committed:
Merge branch 'gh-pages' into ep03-pip

vinisalazar committed:
_episodes/03-packaging-installing.md: moving callout block and edit phrasing.
  - Code review for #79.
  - Move the 'Relative file paths' callout block.
  - Edit phrasing to properly introduce Pip.
